packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -265,6 +268,12 @@ export interface ITestContainerConfig {
 
 	/** Loader options for the loader used to create containers */
 	loaderProps?: Partial<ILoaderProps>;
+
+	/**
+	 * The minVersionForCollab passed to the ContainerRuntime when instantiating it.
+	 * See {@link @fluidframework/container-runtime#LoadContainerRuntimeParams} for more details on this property.
+	 */
+	minVersionForCollab?: MinimumVersionForCollab | undefined;
```

Comment:
A little surprised to see this config updated. Makes me wonder - why not just put this on `IContainerRuntimeOptions`? Feels like the right place for it and might simplify some of the plumbing around you had to do

Reply:
I actually did it in `IContainerRuntimeOptions` in the first iterations of `minVersionForCollab`, but moved it for a couple reasons:
1. We will most likely want to make `minVersionForCollab` a required param one day. We can't do that if it's in `IContainerRuntimeOptions` since that is an optional object itself.
2. It felt a bit odd for one property inside `IContainerRuntimeOptions` to essentially "override" other properties in that same object. It basically felt a bit overloaded/complicated